podcasts/TranscriptViewController.swift

```diff
@@ -292,19 +292,25 @@ class TranscriptViewController: PlayerItemViewController {
         activityIndicatorView.stopAnimating()
     }
 
+    private var previousEpisodeUUID: String?
```

Comment:
I don't think we need to add this logic to `TranscriptViewController`, we can just reset position on the `update` method, which gets called only when `playbackTrackChanged` is fired